lib/module.js

```diff
@@ -190,15 +190,15 @@ Module._findPath = function(request, paths, isMain) {
   // For each path
   for (var i = 0; i < paths.length; i++) {
     // Don't search further if path doesn't exist
-    const curPath = paths[i];
+    var curPath = paths[i];
```

Comment:
Actually, this line was fine. It was just the other 3 that were causing problems.